### DISCUSSION TITLE
ipq806x编译失败

### DISCUSSION THREAD

**Discussion by zheshifandian** [Boos4721/openwrt#144] [Bugs/问题反馈]
部分日志
  OBJCOPY arch/arm/boot/zImage
  Kernel: arch/arm/boot/zImage is ready
make[4]: Nothing to be done for 'Image'.
  DTC     arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dtb
Error: arch/arm/boot/dts/qcom-ipq8062.dtsi:97.1-6 Label or path nss0 not found
Error: arch/arm/boot/dts/qcom-ipq8062.dtsi:103.1-6 Label or path nss1 not found
FATAL ERROR: Syntax error parsing input tree
make[5]: *** [scripts/Makefile.lib:326: arch/arm/boot/dts/qcom-ipq8062-wg2600hp3.dtb] Error 1
make[4]: *** [Makefile:1359: dtbs] Error 2
make[4]: Leaving directory '/home/zheshifandian/openwrt/build_dir/target-arm_cortex-a15+neon-vfpv4_musl_eabi/linux-ipq806x_generic/linux-5.10.119'
make[3]: *** [Makefile:28: /home/zheshifandian/openwrt/build_dir/target-arm_cortex-a15+neon-vfpv4_musl_eabi/linux-ipq806x_generic/linux-5.10.119/.modules] Error 2
make[3]: Leaving directory '/home/zheshifandian/openwrt/target/linux/ipq806x'
make[2]: *** [Makefile:11: compile] Error 2
make[2]: Leaving directory '/home/zheshifandian/openwrt/target/linux'
time: target/linux/compile#90.89#9.04#99.63
以及
Error: arch/arm/boot/dts/qcom-ipq8065.dtsi:166.1-6 Label or path nss0 not found
Error: arch/arm/boot/dts/qcom-ipq8065.dtsi:172.1-6 Label or path nss1 not found
FATAL ERROR: Syntax error parsing input tree
make[6]: *** [scripts/Makefile.lib:326: arch/arm/boot/dts/qcom-ipq8065-nbg6817.dtb] Error 1
make[6]: *** Waiting for unfinished jobs....
Error: arch/arm/boot/dts/qcom-ipq8065.dtsi:166.1-6 Label or path nss0 not found
Error: arch/arm/boot/dts/qcom-ipq8065.dtsi:172.1-6 Label or path nss1 not found
FATAL ERROR: Syntax error parsing input tree
make[6]: *** [scripts/Makefile.lib:326: arch/arm/boot/dts/qcom-ipq8065-r7800.dtb] Error 1

所有具有nss的dtsi貌似都无法通过编译

**Comment by Boos4721** [comment]
yup. I didn't maintain ipq806x. u can try openwrt by official

**Reply by Gaojianli** [reply]
if the ipq806x is orphan, maybe it should be dropped?
